kbs/deploy-kbs.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -euo pipefail
+# Check if the DEBUG env var is set to true
+if [ "${DEBUG:-false}" = "true" ]; then
+    set -x
+fi
+
+# Compulsory env vars
+: "${KEY_FILE}:?"
+: "${CLUSTER_SPECIFIC_DNS_ZONE}:?"
+
+# Ensure we always clone the kbs code base in the kbs sub directory.
+cd $(dirname "$0")
+
+# Pull the KBS code base if it is not available
+if [ ! -d kbs ]; then
+    git clone https://github.com/surajssd/kbs -b k8s-deploy-ingress-install
+fi
+
+pushd kbs/config/kubernetes
+
+# Set the image to use the latest bits from the KBS repository.
+pushd base
+kustomize edit set image kbs-container-image=quay.io/surajd/kbs:latest
```

Comment:
FIXME: Switch to upstream released image. May be as part of pinning to 0.8.0 release?

Reply:
Added.